apps/golden-sample-app/src/app/theme-switcher/theme-switcher.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Input } from '@angular/core';
+import { ThemeManagerService } from './theme-service';
+
+@Component({
+  selector: 'app-theme-switcher',
+  templateUrl: './theme-switcher.component.html',
+})
+export class ThemeSwitcherComponent {
+  constructor(private themeManagerService: ThemeManagerService) {}
+
+  @Input() theme = ThemeManagerService.THEMES[0];
+
+  themes = ThemeManagerService.THEMES;
```

Comment:
If this is not used anywhere, and only in the HTML, `protected` might be better. Same for `onSelect`.